woocommerce/woocommerce-ios
# [Mobile Payments] Add connected reader view

Ready for review.

Includes
- mocked connected reader (that is not persisted)
- light mode AND dark mode

Does not include:
- any changes to the Settings or "Connect your card reader" screens
- any changes to nav bar title (still says "Card Readers" - should say "Manage Card Reader"
- the final form of the modals for "Scanning for readers" or "Found reader" or "Connecting to reader" - for now simple Alerts are being used. FancyAlerts that match the designs will be added in a subsequent PR. This was done to keep this PR under 400 lines.
- unit tests - those will come when we integrate with the CardReaderService in a subsequent PR

To test:
- Navigate to Settings (Gear), then Manage Card Readers
- Tap on "Connect card reader" button
- A "Scanning for Readers" alert will appear. Wait 3 seconds.
- A "Found reader" alert will replace it. Click on "Connect to Reader." Wait 3 seconds.
- A "Connecting to Reader" alert will replace it. Wait 3 seconds.
- A "Connected Readers" list will appear with the mocked reader. Note that its battery level declines 1% every 3 seconds.
- Click on the Disconnect button. You'll be returned to the Connect card reader view.

A light mode movie and a dark mode still:

<img src="https://user-images.githubusercontent.com/1595739/109572115-8a16b780-7aa1-11eb-9c7d-7db38f57273a.gif" width=40%/>

<img src="https://user-images.githubusercontent.com/1595739/109572189-a9ade000-7aa1-11eb-9edf-0b0921550f97.PNG" width=40%/>

Update release notes:
